api/src/test/java/marquez/db/DbTestUtils.java

```diff
@@ -0,0 +1,243 @@
+package marquez.db;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+import static marquez.Generator.newTimestamp;
+import static marquez.common.models.ModelGenerator.newContext;
+import static marquez.common.models.ModelGenerator.newDatasetName;
+import static marquez.common.models.ModelGenerator.newDescription;
+import static marquez.common.models.ModelGenerator.newExternalId;
+import static marquez.common.models.ModelGenerator.newLocation;
+import static marquez.common.models.ModelGenerator.newNamespaceName;
+import static marquez.common.models.ModelGenerator.newOwnerName;
+import static marquez.db.models.ModelGenerator.newRowUuid;
+import static marquez.service.models.ModelGenerator.newDbTableMeta;
+import static marquez.service.models.ModelGenerator.newDbTableMetaWith;
+import static marquez.service.models.ModelGenerator.newJobMetaWith;
+import static marquez.service.models.ModelGenerator.newRunMeta;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Stream;
+import marquez.common.Utils;
+import marquez.common.models.DatasetId;
+import marquez.common.models.JobName;
+import marquez.common.models.NamespaceName;
+import marquez.common.models.RunState;
+import marquez.db.models.DatasetRow;
+import marquez.db.models.ExtendedDatasetVersionRow;
+import marquez.db.models.ExtendedJobVersionRow;
+import marquez.db.models.ExtendedRunRow;
+import marquez.db.models.JobContextRow;
+import marquez.db.models.JobRow;
+import marquez.db.models.JobVersionRow;
+import marquez.db.models.NamespaceRow;
+import marquez.db.models.OwnerRow;
+import marquez.db.models.RunArgsRow;
+import marquez.db.models.RunRow;
+import marquez.service.models.Dataset;
+import marquez.service.models.DbTableMeta;
+import marquez.service.models.Job;
+import marquez.service.models.JobMeta;
+import marquez.service.models.Run;
+import marquez.service.models.RunMeta;
+import org.jdbi.v3.core.Jdbi;
+
+/** Static utility methods for inserting and interacting with rows in the database. */
+final class DbTestUtils {
```

Comment:
There's some overlap here with the `LineageTestUtils` class I created. It's much coarser grained than this, but at some point, we should try to consolidate test utilities.

Reply:
I've opened the issue https://github.com/MarquezProject/marquez/issues/1274 as a follow task 👍